### ISSUE TITLE
Namespace the templates to this module via a folder

### ISSUE THREAD

**Issue by henrahmagix** [incuna/angular-input-fields#4]
`templates/type/my-input.html` should be `templates/angular-input-fields/type/my-input.html` so that a project with a templates path of `type/` doesn't get clobbered by this library, or vice versa: this library doesn't clobber a project's `type/` templates path.

This will be a breaking change.

**Comment by henrahmagix** [comment]
This should complete the filesystem part of the issue:
```shell
mkdir src/templates/angular-input-fields
git mv src/templates/* src/templates/angular-input-fields/
```

And then find-replace `s/templates\/type/templates\/angular-input-fields\/type/g`

**Comment by grahamgilchrist** [comment]
Good idea

**Comment by wytrych** [comment]
@grahamgilchrist It would really nice this one

**Comment by henrahmagix** [comment]
Fixed by #16

@wytrych If you edit a pull-request's description (first comment) and put `Fix #4` in there, it will automatically close this issue once the pull-request is merged (if it's being merged into the base branch).
